tutorials/evals/evaluate_rag_haystack.ipynb

```diff
@@ -0,0 +1,1690 @@
+{
```

Comment:
<p>don't need the last three pentelemetry-sdk opentelemetry-exporter-otlp arize-phoenix-otel</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>